vm/systemSmartContracts/delegation.go

```diff
@@ -478,6 +486,31 @@ func (d *delegation) changeServiceFee(args *vmcommon.ContractCallInput) vmcommon
 	return vmcommon.Ok
 }
 
+func (d *delegation) setCheckCapOnReDelegateRewards(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
```

Comment:
You can improve the code coverage in this function in order to get a green result from the code coverage tool

Reply:
this was already tested.

Reply:
Not all branches inside are tested. Some if branches can be easily tested in unit tests a few lines long.